perl-Digest-MD4/PKGBUILD

```diff
@@ -3,13 +3,13 @@
 _realname=Digest-MD4
 pkgname=perl-${_realname}
 pkgver=1.9
-pkgrel=5
+pkgrel=6
 pkgdesc="Perl Module: Perl interface to the MD4 Algorithm."
 arch=('any')
 license=('PerlArtistic')
 url="https://metacpan.org/${_realname}/"
 groups=('perl-modules')
-makedepends=('libcrypt-devel')
+makedepends=('perl-devel' 'libcrypt-devel')
```

Comment:
I left libcrypt-devel in this one, because Digest-MD4 is old enough that it could actually be using libcrypt.  I should look and see if this is the case or not.

Reply:
Nope.  Will clean that up.